docs/parameters.md

```diff
@@ -137,6 +140,18 @@ Workflow options specific to genomic-medicine-sweden/nallo
 | `extra_paraphase_options` | Extra options to Paraphase, used for test profile. | `string` |  |  | True |
 | `extra_hifiasm_options` | Extra options to hifiasm, used for test profile. | `string` |  |  | True |
 | `extra_yak_options` | Extra options to yak, used for test profile. | `string` |  |  | True |
+| `genmod_compound_snv_penalty` | Genmod compound penalty for SNVs. | `integer` | 6 |  |  |
```

Comment:
Many parameters, but maybe this is the cleanest way to do it. The alternative would be to only have a "single" and "trio" value I guess. I think the current approach is fine though.